src/app/core/services/api.service.ts

```diff
@@ -35,7 +36,7 @@ export class ApiService {
         clone.args = [[['id', '=', 1]]];
         return clone;
       },
-      postProcessor(res: any) {
+      postProcessor(res: User[]) {
```

Comment:
I am not sure it is correct. Maybe `any` is proper rather than `User[]`.
I guess the below api docs tell us it may have any of `object,` `array` and `integer`.
![image](https://user-images.githubusercontent.com/22980553/128556724-61653fc1-5a5f-49ca-9476-3965dbf261dd.png)

Reply:
Okay, so I talked to Waqar, it seems you can specify additional parameters in query api and get a single object. It's not really used on frontend and not currently supported in typings. I'll improve them in future PRs.